modules/sagemaker/sagemaker-groundtruth/settings.py

```diff
@@ -0,0 +1,94 @@
+"""Defines the stack settings."""
+
+from abc import ABC
+from typing import Dict, Optional
+
+from pydantic import Field, computed_field
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class CdkBaseSettings(BaseSettings, ABC):
+    """Defines common configuration for settings."""
+
+    model_config = SettingsConfigDict(
+        case_sensitive=False,
+        env_nested_delimiter="__",
+        protected_namespaces=(),
+        extra="ignore",
+        populate_by_name=True,
+    )
+
+
+class SeedFarmerParameters(CdkBaseSettings):
+    """Seedfarmer Parameters.
+
+    These parameters are required for the module stack.
+    """
+
+    model_config = SettingsConfigDict(env_prefix="SEEDFARMER_PARAMETER_")
+
+    repo_name: str = Field("aiops-to-greengrass-cs")
+    branch_name: str = Field("main")
+    pipeline_assets_prefix: str = Field("pipeline/labeling")
+    use_private_workteam_for_labeling: bool = Field(False)
+    use_private_workteam_for_verification: bool = Field(False)
+    max_labels_per_labeling_job: int = Field(200)
+    labeling_job_private_workteam_arn: Optional[str] = Field(
+        "arn:aws:sagemaker:eu-west-1:0000000000000:workteam/private-crowd/GT1"
+    )
+    verification_job_private_workteam_arn: Optional[str] = Field(
+        "arn:aws:sagemaker:eu-west-1:0000000000000:workteam/private-crowd/GT1"
+    )
+    labeling_pipeline_schedule: str = Field("cron(0 12 1 * ? *)")
+    feature_group_name: str = Field("tag-quality-inspection")
+    model_package_group_name: str = Field("TagQualityInspectionPackageGroup")
+    model_package_group_description: str = Field(
+        "Contains models for quality inspection of metal tags"
+    )
+    repoType: str = Field("CODECOMMIT")
```

Comment:
We'll get type checking for free if [we use an enum](https://docs.pydantic.dev/latest/api/standard_library_types/#enum):

```python
from enum import Enum

class RepoEnum(str, Enum):
    code_commit = 'CODECOMMIT'
    github = 'GITHUB'
```

```suggestion
    repoType: RepoEnum = RepoEnum.code_commit
```